src/test/java/org/openrewrite/kotlin/tree/FieldAccessTest.java

```diff
@@ -114,4 +121,28 @@ void qualifier() {
           )
         );
     }
+
+    @Test
+    void propertyFieldType() {
+        rewriteRun(
+          spec -> spec.recipe(toRecipe(() -> new KotlinIsoVisitor<>() {
+              @Override
+              public J visitFieldAccess(J.FieldAccess fieldAccess, ExecutionContext ctx) {
+                  if (fieldAccess.getSimpleName().equals("MIN_VALUE")) {
+                      JavaType.Variable fieldType = fieldAccess.getName().getFieldType();
+                      assertThat(fieldType).isNotNull();
+                      assertThat(fieldType.getName()).isEqualTo("MIN_VALUE");
+                      assertThat(fieldType.getType().toString()).isEqualTo("kotlin.Int");
+                      assertThat(fieldType.getOwner().toString()).isEqualTo("kotlin.Int$Companion");
+                  }
+                  return super.visitFieldAccess(fieldAccess, ctx);
+              }
+          })),
+          kotlin(
+            """
+            val i = Int.MIN_VALUE
+            """
```

Comment:
Indentation nit: indentation of end delimiter of text block should be 2 spaces to the right of start delimiter and code should be left-aligned to that end delimiter.

Reply:
Meaning something like this?
```
"""
  Code here.
  """
```

If so, I'll sweep through when I have time to fix the tests -- the current PR matches the rest of the code